app/services/foreman_ansible/playbooks_importer_helper.rb

```diff
@@ -0,0 +1,72 @@
+module ForemanAnsible
+  class PlaybooksImporterHelper
```

Comment:
I like that you divide things to keep the classes small. I could not determine the pattern why some methods stay in importer while other move to helper, what is the decision key?

Reply:
I thought of helper methods that are not really related to the logic of the importer will be on a different class